src/readii/process/images/crop.py

```
@@ -0,0 +1,449 @@
+    image : sitk.Image
+        Image to validate the new dimensions for.
+    new_dimensions : tuple or int
+        Tuple of values representing the new dimensions to validate, or a single integer representing the number of dimensions.
+
+    Raises
+    ------
+    ValueError
+        If the new dimensions are not valid for the image.
+    """
+    # Check that the number of dimensions in the new dimensions matches the number of dimensions in the image
+    if type(new_dimensions) == tuple:
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)